src/store/actions.js

```diff
@@ -391,6 +392,7 @@ export default {
 				message: {
 					...message.data,
 					body: message.data.isHtml ? body.value : toPlain(body).value,
+					editorBody: body.value,
```

Comment:
this is also wrong and causes https://github.com/nextcloud/mail/pull/6372#issuecomment-1144910893